lib/unit/models/payment/create_book_payment_request.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# Request to create a book payment
+# @see https://docs.unit.co/book-payments#book-payments
+module Unit
+  module Payment
+    class CreateBookPaymentRequest
```

Comment:
instead  of taking a relationship.
lets take accountId and counterpartyAccountId and build the relationship by yourself.